testing/ncids-css-testing/stories/uswds-native/usa-prose/src/usa-prose.twig

```diff
@@ -0,0 +1,100 @@
+<div class="usa-line-length-example">
+  <h3 class="site-preview-heading margin-top-0">Line length</h3>
+  <section class="typography-example usa-prose margin-top-1">
```

Comment:
This section gets flagged by HTML validator as section without a heading.  Make it a div?